unidist/core/backends/mpi/core/controller/api.py

```diff
@@ -49,6 +51,56 @@
 topology = dict()
 # The global variable is responsible for if MPI backend has already been initialized
 is_mpi_initialized = False
+threads = []
+BACKOFF = 0.001
+exitFlag = False
```

Comment:
```suggestion
exit_flag = False
```
Also, put a comment what is this (for)?